src/routes/settings/+page.svelte

```diff
@@ -63,12 +64,14 @@
 	>
 		<div class="flex flex-col gap-4 flex-1">
 			<Settings.Conversion bind:settings />
-			<Settings.Vertd bind:settings />
+			{#if !DISABLE_ALL_EXTERNAL_REQUESTS}
+				<Settings.Vertd bind:settings />
+			{/if}
 		</div>
 
 		<div class="flex flex-col gap-4 flex-1">
 			<Settings.Appearance />
-			{#if PUB_PLAUSIBLE_URL}
+			{#if PUB_PLAUSIBLE_URL && !DISABLE_ALL_EXTERNAL_REQUESTS}
```

Comment:
Disabling the entire privacy & data section wouldn't be ideal, the converter caching management is in this section

you would want to remove the plausible analytics section only in its file